app/src/main/java/ca/klapstein/baudit/models/DataModel.java

```diff
@@ -47,14 +42,21 @@ static public boolean uniqueID(String username) {
      * <p>
      * TODO: implement
      *
-     * @param username {@code String}
-     * @param password {@code String}
+     * considering param type {@code String}
+     * @param username {@code Username}
+     * @param password {@code Password}
      * @return {@code boolean}
      */
-    public boolean validateLogin(String username, String password) {
-        // TODO: implement from RemoteModel
+    public boolean validateLogin(Username username, Password password) {
```

Comment:
Would accepting Username and Password classes increase coupling? As opposed to passing username.getUsernameString and password.getPassword during calls to validateLogin? I recognize though that it makes it a bit cleaner to retrieve the string in validateLogin.